why_postgres.mdx

```diff
@@ -0,0 +1,137 @@
+
+Since then, tens of thousands of developers discovered Xata, actively used it in their applications, and have given us extremely valuable feedback. Developers in our [community](https://xata.io/blog/tags/community) appreciate how easy Xata is to get started with, iterate quickly, and the feeling that it's more like a SaaS product than another DBaaS.
+
+Over the past year, we have continued to release features that show the value of a database platform, as opposed to a DBaaS. From a new git-like [developer workflow](https://xata.io/blog/workflow-github-vercel-netlify-xata) for making schema changes with our branching functionality, to [file attachments](https://xata.io/blog/file-attachments), which make it easy to work with files and images as columns in your database. These features generated a [lot](https://x.com/rauchg/status/1651614599874252802?s=20) of [excitement](https://news.ycombinator.com/item?id=37324370).
+
+Last but not least, we have open sourced [pgroll](https://github.com/xataio/pgroll), a tool that provides zero-downtime and reversible schema migrations for any PostgreSQL database. This has also generated [quite](https://news.ycombinator.com/item?id=37752366) a bit of [interest](https://simonwillison.net/2024/Jan/30/pgroll/) from the broader Postgres community. Later this week we'll be announcing another open source project that we think you'll love 🦎.
+
+## The serverless Postgres service
+
+This brings us to today's launch: a new serverless Postgres service at the center of the Xata platform. Looking at the Postgres service in isolation (ignoring everything else in the Xata platform), it fares pretty well:
+
+- Separation of compute and storage, providing bottomless storage.
+- Optional automatic scaling up and down based on the load.
+- High availability via read replicas in multiple availability zones and automatic failover.
+- Support for both the native Postgres wire protocol and SQL over HTTP (for easy access from edge functions).
+- Support for the most common Postgres extensions.
+- The ability to scale up to 1TB of RAM and 128 vCPUs.
+
+We have designed the Postgres service to optimize for **reliability**, **scalability**, and **cost efficiency**.
+
+#### Reliability
+
+The Xata Postgres service is using AWS Aurora under the hood, a battle-tested service that is used by many very large Postgres installations. We have been using AWS Aurora since the initial Xata launch and we have a reliability [track record](https://www.xatastatus.com/) that we are proud of. We now also have an official [SLA](/docs/appendix/sla).
```

Comment:
```suggestion
The Xata Postgres service is using AWS Aurora under the hood, a battle-tested service that is used by many very large Postgres installations. We have been using AWS Aurora since the initial Xata launch and we have a reliability [track record](https://www.xatastatus.com/) that we are proud of. We now also have an official [SLA](/docs/appendix/sla).
```
```suggestion
The Xata Postgres service is using AWS Aurora under the hood, a battle-tested service that is used by many very large Postgres installations. We have been using AWS Aurora since the initial Xata launch and we have a reliability [track record](https://www.xatastatus.com/) that we are proud of. We now also have an official [SLA](https://xata.io/docs/appendix/sla).
```

Reply:
This shouldn't be needed once it's merged, the relative docs links should work.